atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -125,11 +127,41 @@ def run(self):
                     filepath = os.path.join(filedir, f)
                     archive.write(os.path.join(root, f), filepath, zipfile.ZIP_DEFLATED)
             manifest_files = archive.namelist()
-            if not manifest_files:
-                self.log.error('Empty operator manifests directory')
-                raise RuntimeError('Empty operator manifests directory')
             self.log.debug("Archiving operator manifests: %s", manifest_files)
 
         shutil.rmtree(manifests_path)
 
         return manifests_zipfile_path
+
+    def _verify_csv(self, manifests_path):
+        """Verify the CSV file from the built image
+
+        :param str manifests_path:
+        :raises: ValueError if more than one CSV files are found, or the single
+            CSV is different from the one in the repo (compared by hash digest).
+        """
+        try:
+            image_csv = OperatorManifest.from_directory(manifests_path).csv
+        except ValueError as e:
+            if 'Missing ClusterServiceVersion' in str(e):
+                raise ValueError('No ClusterServiceVersion in output operator bundle.')
+            if 'Operator bundle may contain only 1 CSV file' in str(e):
+                raise ValueError(
+                    'The output operator bundle contains more than one CSV files, '
+                    'but only 1 should be included.')
```

Comment:
Can we have something like this instead searching in error messages?

```suggestion
            raise ValueError("Operator manifests check in built image failed: {e}")
```